duckduckgo/macos-browser
# Show http/s protocol in the home page URL

Task/Issue URL: https://app.asana.com/0/inbox/1204051006248670

**Description**:
Show the URL protocol in the homepage settings view

**Steps to test this PR**:
1. Go to settings > general and set a new home page
2. Observe the protocol (https) is displayed in the settings view

<img width="1213" alt="Screenshot 2023-11-02 at 12 29 22 PM" src="https://github.com/duckduckgo/macos-browser/assets/1156669/19a00a67-587c-46d4-afb7-77541050b4af">




---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
